Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj

```diff
@@ -12,10 +12,11 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
     <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
+    <TargetPlatformMinVersion>10.0.14393.0</TargetPlatformMinVersion>
```

Comment:
Would it not be possible to target `10.0.10240.0` in `TargetPlatformMinVersion` instead of `10.0.14393.0`? I have projects that target this version, which is LTSC, and are still in XF 2.5 for that reason.

Reply:
@YZahringer for this one we're going to just do `14393` because we've already done the leg work for that change. Can you open a new issue around `10240` and then we can discuss there going down to a lower version?

Reply:
NM @YZahringer  I see that you added some additional comments in this PR

I moved those over here
https://github.com/xamarin/Xamarin.Forms/issues/8570

So we can discuss and look into lowering again after this pr is merged